training/05_updating_a_data_package.Rmd

```diff
@@ -15,7 +15,7 @@ What if the researcher notices that some information needed to be updated in the
 If you haven't already:
 
 * Locate the data package you published in the [previous exercise](#exercise-3b) by navigating to the URL test.arcticdata.io/#view/...
-* Load the package and EML into R using the above commands.
+* Load the package and EML into R using the above commands (chapter 5.21).
```

Comment:
hint - you can link to other sections in the r-markdown similar to how the link was used here `[previous exercise](#exercise-3b)`. I'll let you play around with it but let me know if you get stuck